src/app/Recordings/ArchivedRecordingsTable.tsx

```diff
@@ -457,3 +505,12 @@ export const ArchivedRecordingsTable: React.FunctionComponent<ArchivedRecordings
     </Drawer>
   );
 };
+
+export const convertToNumber = (recordingName: string): number => {
```

Comment:
Recording rows require a numeric index. 

I used `id` for ActiveRecording. However, ArchivedRecording does not so I used its unique and calculate its hash. Kinda a hack but it should do a good job avoiding collision. Any better suggestions?

Reply:
Makes sense to me. The archived recording name is unique in the context of an individual target, so hashing the name seems like a good solution.

Reply:
Could you extract this into somewhere within `src/app/utils` and rename it something like `hashcode`?